src/components/ui/03-organisms/images-grid.jsx

```diff
@@ -1,37 +1,114 @@
-import * as React from "react";
+import { useState } from "react";
 import ImageList from "../02-molecules/image-list";
+import ImagePreview from "../02-molecules/image-preview";
 import ImageControls from "../02-molecules/image-controls";
+import { addApolloState, initializeApollo } from '../../../lib/apolloClient';
+import UserSubmission from '../../../graphql/user-submission.graphql';
```

Comment:
It seems this import is making the build failing

Reply:
In fact, I think we don't have need that graphql file like that, since we are just getting the image there, we need the query in this way:

`query {
  nodeQuery {
    entities {
      entityType
      ... on NodeUserSubmission {
        title
        fieldUserEmail
        fieldProcessedImage1 {
          entity {
            mid
            entityRendered
            thumbnail {
              alt
              targetId
            }
          }
        }
      }
    }
  }
}`